unit-tests/py/rspy/devices.py

```diff
@@ -718,15 +718,15 @@ def get_phys_port(dev):
         for opt,arg in opts:
             if opt in ('--list'):
                 action = 'list'
-            elif opt in ('--port'):
+            elif opt in ('--port','--PORT'):
                 if not hub:
                     log.f( 'No hub available' )
                 all_ports = hub.all_ports()
                 str_ports = arg.split(',')
                 ports = [int(port) for port in str_ports if port.isnumeric() and int(port) in all_ports]
                 if len(ports) != len(str_ports):
                     log.f( 'Invalid ports', str_ports )
-                hub.enable_ports( ports, disable_other_ports=False )
+                hub.enable_ports( ports, disable_other_ports=(opt == '--PORT') )
```

Comment:
Why?
If we are here opt is ether --port or --PORT.
Here you set a different behavior for uppercase and lower case, right?

Reply:
Yes, I wanted the ability to automatically disable other ports without have to use --none and then --port (they also require a delay in between). They both enable a specific port, but differ in how they treat other ports.

Reply:
I think this deserve a comment, otherwise it looks like a copy paste BUG or something

Reply:
Not sure how it looks like a copy paste bug... `disable_other_ports=(opt == '--PORT')` literally says "disable other ports if the --PORT was specified and not --port".
But OK, I'll add that comment.